tests/ui/uninlined_format_args_2021.rs

```diff
@@ -0,0 +1,23 @@
+// run-rustfix
+// edition:2021
```

Comment:
This is the default edition, so this shouldn't be required here. But that's not keeping me from r+ing it.